src/style.css

```diff
@@ -1,3 +1,82 @@
+html {
```

Comment:
Let's avoid setting that rule at the -html level. It will do as a rule for header, for example.